src/App.js

```diff
@@ -1,57 +1,43 @@
-import React from 'react';
-import logo from './logo.svg';
-import { Counter } from './features/counter/Counter';
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+/* import { useSelector, useDispatch } from 'react-redux';
+import { getWeather } from './features/wheather/weatherSlice'; */
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.